magic/Magdir/apple

```diff
@@ -424,7 +424,13 @@
 #>0x410	string	disk\ image	UDIF read/write image (UDRW)
 
 # From: Toby Peterson <toby@apple.com>
```

Comment:
if this is really how it was committed, then this comment is misplaced. it should comment the bplist00 entry